tools/wptrunner/wptrunner/formatters/chromium.py

```diff
@@ -121,10 +121,14 @@ def _store_test_result(self, name, actual, expected, actual_metadata,
         cur_dict = self.tests
         for name_part in name_parts:
             cur_dict = cur_dict.setdefault(name_part, {})
-        cur_dict["actual"] = actual
+        statuses = cur_dict.get("actual", "").split()
+        statuses.append(actual)
+        cur_dict["actual"] = " ".join(statuses)
         cur_dict["expected"] = expected
         if duration is not None:
-            cur_dict["time"] = duration
+            cur_dict.setdefault("time", duration)
```

Comment:
This is just `cur_dict["time"] = duration` I think?

Reply:
The old line overwrite `time` with the duration of the last run, when it should be the first. From the linked document:

> Optional. If present, the time it took in seconds to execute the first invocation of the test.

Added a comment. Updating PR description as well.